pep-0678.rst

```diff
@@ -229,6 +245,49 @@ proposed ``__note__`` semantics, but this would be rarely and inconsistently
 applicable.
 
 
+Augment the ``raise`` statement
+-------------------------------
+One discussion proposed ``raise Exception() with "note contents"`` , but this
+does not address the original motivation of compatibility with ``ExceptionGroup`` .
+
+Furthermore, this would be an additional feature over adding the attribute and
+modifying the traceback-formatting code, and we do not expect ``__note__`` to be
+so frequently-used as to justify new syntax or even a new ``.with_note()`` method
+(analogous to ``.with_traceback()`` ).
+
+
+Allow any object, and cast to string for display
+------------------------------------------------
+We have not identified any scenario where libraries would want to do anything but either
+concatenate or replace notes, and so the additional complexity and interoperability
+challenges do not seem justified.
+
+Variations on this have also been proposed for forward-compatibility with a future
+structured API, but in the absence of any proposed use-case (see also "Non-goals")
+we strongly prefer to keep things simple - and note that implementing any nontrival
+future behavior would then break backwards-compatibility.
```

Comment:
Besides awkwardness in the syntax, its unclear to me whether this refers to what would happen if you retain the current proposal of only allowing `str` rather than casting to it (which if so, would seem to be somewhat redundant with the first phrase), or is saying that any non-trivial future behavior implemented in a future structured API would break backward compat anyway.